Add testplan for editing NPC and minigame description (#138)
Gamify-IT/docs: dev-manuals/test-plans/lecturer-interface/u-lecturer-interface-20-edit-minigame-description.md

# Lecturer-Interface: Edit minigame description (`u.lecturer-interface-20`)

Version: V1.0, 29.08.2022 \
Author: Max Kästner

## Description

> The description of a specific minigame can be changed on its view.

Reviewer: ```suggestion
The description of a specific minigame can be changed.
```

## Precondition

At least one course is created. The tester is located where the minigames of an area are listed.

## Postcondition

> The description of the minigame was changed to the new entered description.

Reviewer: ```suggestion
The description of the minigame was changed to the newly entered description.
```

## Typical procedure

1. Click the edit description button of a minigame
2. A input field shows up
3. Change the minigame description to a new description in the input field

> 4. Hit the submit button

Reviewer: ```suggestion
4. Submit your changes, i.e. through the corresponding button
```

> 5. A feedback toast message appears which contains that the minigame description was updated
> 6. The new minigame description replaces the old minigame description in the view and the input field disappears

Reviewer: ```suggestion
5. A "Success" feedback message appears
6. The new minigame description replaces the old minigame description
7. The input field disappears
```

## Alternative procedures

> 4.1: Hit the cancel button: \
>     4.1.1 A feedback toast message appears which contains that minigame description was not updated \
>     4.1.2 The minigame description was not updated in the view and the input field disappears 

Reviewer: ```suggestion
4.1: Cancel your submission: \
    4.1.1 A "Changes cancelled" feedback message appears \
    4.1.2 The minigame description was not updated \
    4.1.3 The input field disappears 
```
